github/pulls_reviews.go

```diff
@@ -5,15 +5,192 @@
 
 package github
 
-import "time"
+import (
+	"fmt"
+	"time"
+)
 
 // PullRequestReview represents a review of a pull request.
 type PullRequestReview struct {
-	ID          *int       `json:"id,omitempty"`
-	User        *User      `json:"user,omitempty"`
-	Body        *string    `json:"body,omitempty"`
-	SubmittedAt *time.Time `json:"submitted_at,omitempty"`
+	ID             *int       `json:"id,omitempty"`
+	User           *User      `json:"user,omitempty"`
+	Body           *string    `json:"body,omitempty"`
+	SubmittedAt    *time.Time `json:"submitted_at,omitempty"`
+	CommitID       *string    `json:"commit_id,omitempty"`
+	HTMLURL        *string    `json:"html_url,omitempty"`
+	PullRequestURL *string    `json:"pull_request_url,omitempty"`
 
-	// State can be "approved", "rejected", or "commented".
+	// State can be "ACCEPTED", "DISMISSED", "CHANGES_REQUESTED" or "COMMENTED".
 	State *string `json:"state,omitempty"`
 }
+
+func (p PullRequestReview) String() string {
+	return Stringify(p)
+}
+
+// PullRequestReviewComment represents a comment left on a pull request review
+type PullRequestReviewComment struct {
+	ID             *int       `json:"id,omitempty"`
+	User           *User      `json:"user,omitempty"`
+	Body           *string    `json:"body,omitempty"`
+	CreatedAt      *time.Time `json:"created_at,omitempty"`
+	UpdatedAt      *time.Time `json:"updated_at,omitempty"`
+	CommitID       *string    `json:"commit_id,omitempty"`
+	HTMLURL        *string    `json:"html_url,omitempty"`
+	PullRequestURL *string    `json:"pull_request_url,omitempty"`
+}
+
+// DraftReviewComment represents a comment part of the review
+type DraftReviewComment struct {
+	Path     *string `json:"path,omitempty"`
+	Position *int    `json:"position,omitempty"`
+	Body     *string `json:"body,omitempty"`
+}
+
+// PullRequestReviewRequest represents a request to create a review
+type PullRequestReviewRequest struct {
+	Body     *string              `json:"body,omitempty"`
+	Event    *string              `json:"event,omitempty"`
+	Comments []DraftReviewComment `json:"comments,omitempty"`
+
+	// Message is used (optionally) while dismissing a review
+	Message *string `json:"message,omitempty"`
```

Comment:
Hmm... I think it's better to create another struct like `PullRequestReviewDissmisalRequest` or `PullRequestReviewDissmisal` because the format is completely different.